test/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java

```diff
@@ -98,7 +108,14 @@ public void successfulDelete() {
         doReturn(survey).when(mockSurveyDao).getSurvey(any());
         
         service.deleteSurvey(survey);
-        
+
+        // verify query args
+        ArgumentCaptor<String> queryCaptor = ArgumentCaptor.forClass(String.class);
+        verify(mockSharedModuleMetadataService).queryAllMetadata(anyBoolean(), anyBoolean(), queryCaptor.capture(),  anySetOf(String.class));
+
+        String queryStr = queryCaptor.getValue();
+        assertEquals(queryStr, "surveyGuid=\'" + survey.getGuid() + "\' AND surveyCreatedOn=" + survey.getCreatedOn());
```

Comment:
Minor: You have this backwards. In JUnit, the expected comes before the actual. (Here and in successfulDeletePermanently().)